src/pudl/metadata/resources/eia923.py

```diff
@@ -754,6 +757,32 @@
         "sources": ["eia923"],
         "etl_group": "eia923",
     },
+    "core_eia923__monthly_energy_storage": {
+        "description": TABLE_DESCRIPTIONS["core_eia923__monthly_energy_storage"],
+        "schema": {
+            "fields": [
+                "plant_id_eia",
+                "report_date",
+                "prime_mover_code",
+                "energy_source_code",
+                "data_maturity",
```

Comment:
Wasn't sure if this was one of the columns to keep or drop

Reply:
I guess I was thinking that an output table could pull plant name data from a harvested EIA entity table and `data_maturity` should not be a harvested value (at least between 923 and 860). I'm not sure where the `data_maturity` source of truth lives for each data source. I can look into it.

Reply:
`data_maturity` is inserted by us on a record-by-record basis when we read the data in from the spreadsheets, based on what kind of spreadsheet we're reading (final, early release, monthly updates, etc)

Reply:
It looks like `data_maturity` is in all the `core` table schemas so I'm going to leave it there